### ISSUE TITLE
SwizzleFunctionBuilder returns SwizzlePair when arguments passed only one SwizzlePair

### ISSUE THREAD

**Issue by 0x0c** [MarioIannotta/SwizzleSwift#1]
In `SwizzleSwiftDemo`, Xcode cannot build the following code because of an error `Cannot convert value of type 'SwizzlePair' to closure result type '[SwizzlePair]'.`
```swift
Swizzle(ViewController.self) {
    #selector(viewDidLoad) ==> #selector(myViewDidLoad)
}
```

Passing an array of `SwizzlePair` instead of `@_functionBuilder` may resolve this issue.

```swift
Swizzle(ViewController.self, [
    #selector(viewDidLoad) ==> #selector(myViewDidLoad),
    #selector(viewWillAppear(_:)) ==> #selector(myViewWillAppear(_:)),
])
```

**Comment by MarioIannotta** [comment]
This has been fixed in [1.0](https://github.com/MarioIannotta/SwizzleSwift/releases/tag/1.0).
Now you can Swizzle a pair of methods or more with the same api.

```swift
Swizzle(ViewController.self) {
    #selector(viewDidLoad) <-> #selector(myViewDidLoad)
}
```

```swift
Swizzle(ViewController.self) {
    #selector(viewDidLoad) <-> #selector(myViewDidLoad),
    #selector(viewWillAppear(_:)) <-> #selector(myViewWillAppear(_:))
}
```

**Comment by 0x0c** [comment]
Thank you :)
